src/boot/efi/pe.c

```diff
@@ -141,6 +142,7 @@ static size_t section_table_offset(const DosFileHeader *dos, const PeFileHeader
 }
 
 static void locate_sections(
+                const uint8_t *base,
```

Comment:
we generally want that pointers to "blobs", i.e. for us "unstructured data" are of type `void*`